metamorph/lib/support_functions.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/python
+import logging
+import logging.config
+import os.path
+import json
+
+
+class ExistingMetamorphMetadataException(Exception):
+    pass
```

Comment:
My original thought was to create custom exceptions which can be handled afterwards, but they are not now ...